Frends.OAuth.ParseToken/Frends.OAuth.ParseToken/ParseToken.cs

```diff
@@ -27,9 +27,10 @@ public class OAuth
     /// <param name="input">Input parameters</param>
     /// <param name="options">Option parameters</param>
     /// <param name="cancellationToken">Token generated by Frends to stop this task.</param>
-    /// <returns>Object { ClaimsPrincipal ClaimsPrincipal, SecurityToken Token }</returns>
+    /// <returns>Object { string Id, string Issuer, string Audiences, string SecurityKeyId, string SigningKeyId, DateTime ValidFrom, DateTime ValidTo }</returns>
     public static async Task<ParseResult> ParseToken([PropertyTab] Input input, [PropertyTab] Options options, CancellationToken cancellationToken)
     {
+        var tokenResult = new List<TokenResult>();
```

Comment:
You don't need to create a list, since you are only returning one result, not multiple results.